prisma/schema.prisma

```diff
@@ -9,3 +6,11 @@ datasource db {
   provider = "sqlite"
```

Comment:
Interesting decision. Could you give pros and cons of choosing `sqlite` over `postgresql`?